invoice/src/main/java/org/killbill/billing/invoice/usage/SubscriptionUsageInArrear.java

```diff
@@ -164,6 +176,7 @@ public UsageKey apply(final Usage input) {
                                        new ContiguousIntervalConsumableUsageInArrear(usage, accountId, invoiceId, rawSubscriptionUsage, existingTrackingIds, targetDate, rawUsageStartDate, usageDetailMode, internalTenantContext);
 
                     inFlightInArrearUsageIntervals.put(usageKey, existingInterval);
+                    allSeenUnitTypes.addAll(existingInterval.getUnitTypes());
```

Comment:
Shouldn't this be reset for each billing event though?

Let's say we have 2 billing events with different usage types:
*` efftDt1`: b1 {t1, t2}
* `efftDt2`: b2 {t2}

->  If we record usage type t1, after `efftDt2` -> park?